### ISSUE TITLE
Plex Integration - how are y'all doing?

### ISSUE THREAD

**Issue by bwt615** [meeb/tubesync#455]
Does anyone have a consistently working metadata solution for Plex? Specifically for things like reoccurring news shows. I havent been able to get zero YouTube working consistently and the issues over there are dead so I thought I'd start a new thread over here for 24. There's an older thread here that I posted on and didn't get a response so thought I'd start a new one.

**Comment by wormhoudt** [comment]
As far as I can tell there isn't a way to make this work. I have tried every combination of file and folder path, NFO files with XBMCnfoTVImporter ... none of it seems to work. Even if the file is named in the format that plex wants it still won't use the episode title from the file name. 

I think folder view is the only option here. I'd love to be wrong tho.

**Comment by bwt615** [comment]
Me too!! There has to be a way. Tell me more - what's the folder experience like?

**Comment by meeb** [comment]
Last I looked into this it required a plugin to get the data from somewhere and stuff it into Plex (e.g. like how XBMCnfoTVImporter is meant to work). Naturally Plex nerfing plugins and making them a pain to install significantly hinders this.

I guess I could have a look at having TubeSync spam update fields directly via the API. These fields:

https://www.plexopedia.com/plex-media-server/api/library/movie-update/#fields

That might at least set the title and date properly. That's unfortunately for "movie" libraries only though, AFAIK the only way to get "tv" or "series" style libraries to update items correctly would be to write a plugin or a custom metadata provider unless I've missed something in the API.

**Comment by bwt615** [comment]
Zero tv series is the one I've seen working - have had limited luck getting it working. Haven't played at all with the metadata route

**Comment by garbled1** [comment]
I have it working with zeroiq plex.  The trick is to name your dirs like:
'Hand_Tool_Rescue [youtube2-UCasG9kJWi1eVxM0QkyqKVJQ]'

and then my files:
'20231217 - rare-bow-chainsaw-restoration [Wjv75kJyZ7M].info.json'
'20231217 - rare-bow-chainsaw-restoration [Wjv75kJyZ7M].jpg'
'20231217 - rare-bow-chainsaw-restoration [Wjv75kJyZ7M].mkv'
'20231217 - rare-bow-chainsaw-restoration [Wjv75kJyZ7M].nfo'

**Comment by bwt615** [comment]
Nice! You had a few  differences from me - the use of the title in the file, external metadata files and use of youtube2- in the channel. Trying to see if it will work with your setup!

**Comment by bwt615** [comment]
> I have it working with zeroiq plex. The trick is to name your dirs like: 'Hand_Tool_Rescue [youtube2-UCasG9kJWi1eVxM0QkyqKVJQ]'
> 
> and then my files: '20231217 - rare-bow-chainsaw-restoration [Wjv75kJyZ7M].info.json' '20231217 - rare-bow-chainsaw-restoration [Wjv75kJyZ7M].jpg' '20231217 - rare-bow-chainsaw-restoration [Wjv75kJyZ7M].mkv' '20231217 - rare-bow-chainsaw-restoration [Wjv75kJyZ7M].nfo'

This is immediately working better for me - I think it would be great to get this documented as a working method in either readme!

**Comment by ZeroQI** [comment]
added example to readme

**Comment by garbled1** [comment]
yeah, once I set everything to "youtube2" I was immediately happier with the zeroiq integration.  It solved all my problems.  Glad it worked for you too!

**Comment by meeb** [comment]
This seems resolved for now so I'll close it. Please open a new issue if you have further questions.
